pkg/trimaran/resourcestats.go

```diff
@@ -195,24 +195,24 @@ func GetNodeRequestsAndLimits(podInfosOnNode []*framework.PodInfo, node *v1.Node
 			// get requests and limits for pod
 			requested = GetResourceRequested(p)
 			limits = GetResourceLimits(p)
-			// make sure limits not less than requests
-			SetMaxLimits(requested, limits)
 		}
-		reqCpu := requested.MilliCPU
-		reqMem := requested.Memory
-		limitCpu := limits.MilliCPU
-		limitMem := limits.Memory
+		// make sure limits not less than requests
+		SetMaxLimits(requested, limits)
```

Comment:
No need to take `SetMaxLimits()` outside loop as the last element in the list, namely `pod` had already passed that check in [CreatePodResourcesStateData()](https://github.com/kubernetes-sigs/scheduler-plugins/blob/6df71a420673837e2aaf90540f67d3cac18a799a/pkg/trimaran/lowriskovercommitment/lowriskovercommitment.go#L261).

Reply:
Sure @atantawi , reversed this change.